product.yml

```diff
@@ -1262,3 +1262,5 @@ bug_mapping:
       issue_component: GitOps ZTP
     ose-aws-ecr-image-credential-provider:
       issue_component: Cloud Compute / Cloud Controller Manager
+    kube-compare-artifacts:
+      issue_component: CDN Repo Mapping
```

Comment:
This cannot be right

Reply:
From the ticket: bug_component: oc / cluster-compare